src/core/statement.cpp

```diff
@@ -682,28 +644,42 @@ void statement_impl::describe()
         switch (dtype)
         {
         case dt_string:
-            bind_into<dt_string>();
-            break;
         case dt_blob:
-            bind_into<dt_string>();
-            break;
         case dt_xml:
-            bind_into<dt_string>();
+        {
+            std::pair<std::string*, indicator*> holder = row_->alloc_data_holder_string();
+            exchange_for_row(into(*holder.first, *holder.second));
```

Comment:
Instead of repeating this several times, would it make sense to add some helper function taking this `holder` directly and allowing to write something like this instead:
```suggestion
            exchange_for_row_holder(row_->alloc_data_holder_string());
```
?